### ISSUE TITLE
Wrong error of `SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse`

### ISSUE THREAD

**Issue by mcsknp** [slevomat/coding-standard#1286]
The below code return `Type MySpace\MyClass is not used in this file.` error when run phpcs

```php
<?php

declare(strict_types=1);

use MySpace\MyClass;

$a = ['sample' => 'value'];

echo "{$a[MyClass::MY_CONST]}";
```

Class `MyClass` is defined as below
```php
<?php

declare(strict_types=1);

namespace MySpace;

class MyClass
{
    public const MY_CONST = 'sample';
}
```

**Comment by kukulich** [comment]
@mcsknp Could you please test `dev-master` ?
